mon-pix/app/router.js

```diff
@@ -69,7 +69,9 @@ Router.map(function() {
   this.route('fill-in-campaign-code', { path: '/campagnes' });
 
   this.route('campaigns', { path: '/campagnes/:code' }, function() {
-    this.route('start-or-resume', { path: '/' });
+    this.route('entry-point', { path: '/' });
+    this.route('not-found', { path: '/oups' });
```

Comment:
ligne 105, il y a déjà une route "not-found" pour plus de clarté, peut-être qu'on peut appeler celle-ci "campaign-not-found" ?